components/todo/Header.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <header class="header">
     <div class="header__left">
-      <h1>{{ $t('todos') }}</h1>
-      <select @change="setLocale">
+      <ProjectLogo w="34" h="34" />
+      <span class="todo__title">{{ $t('todos') }}</span>
+      <select class="todo__language" @change="setLocale">
```

Comment:
by default english should be selected instead of `Select Language`

Reply:
resolved. Bhai. Thanks